src/components/LayoutFooter/index.tsx

```diff
@@ -1,127 +1,217 @@
 import React from 'react'
 import cn from 'classnames'
-import includes from 'lodash/includes'
 
-import { LayoutModifiers, ILayoutModifiable } from '../MainLayout'
 import LayoutWidthContainer from '../LayoutWidthContainer'
 import Link from '../Link'
-import IconSet from '../IconSet'
-
+import SocialIcon, { ISocialIcon } from '../SocialIcon'
+import ShowOnly from '../ShowOnly'
 import { getFirstPage } from '../../utils/shared/sidebar'
 
-import { ReactComponent as LogoSVG } from '../../../static/img/logo-white.svg'
+import { ReactComponent as LogoSVG } from '../../../static/img/logo.svg'
+import { ReactComponent as GithubSVG } from '../SocialIcon/github.svg'
+import { ReactComponent as TwitterSVG } from '../SocialIcon/twitter.svg'
+import { ReactComponent as DiscordSVG } from '../SocialIcon/discord.svg'
+import { ReactComponent as CmlSVG } from '../../../static/img/cml-icon.svg'
+import { ReactComponent as StudioSVG } from '../../../static/img/studio-icon.svg'
+import { ReactComponent as IterativeSVG } from '../../../static/img/iterative-icon.svg'
+
 import styles from './styles.module.css'
 
 const docsPage = getFirstPage()
 
-const LayoutFooter: React.FC<Required<ILayoutModifiable>> = ({ modifiers }) => (
+interface IFooterLinkData {
```

Comment:
I'm pretty new to TypeScript, please let me know if my typescript code can be improved!